packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -10,13 +10,21 @@ import PropTypes from 'prop-types'
 import {useHistory, useLocation} from 'react-router-dom'
 import {getAssetUrl} from 'pwa-kit-react-sdk/ssr/universal/utils'
 import {getAppOrigin} from 'pwa-kit-react-sdk/utils/url'
-
+import {getConfig} from 'pwa-kit-runtime/utils/ssr-config'
+import {useQueries} from '@tanstack/react-query'
+import {
+    useAccessToken,
+    useCategory,
+    useCommerceApi,
+    useCustomerType
+} from 'commerce-sdk-react-preview'
+import * as queryKeyHelpers from 'commerce-sdk-react-preview/hooks/ShopperProducts/queryKeyHelpers'
```

Comment:
I'm going to create a ticket to properly expose query key helpers as public API...